tensorflow/compiler/xla/service/gpu/kernel_mapping_scheme.h

```diff
@@ -76,19 +76,35 @@ namespace gpu {
 class KernelMappingScheme {
  public:
   enum { DimZ = 0, DimY, DimX, DimTot };
+  // TODO: rename Dilated to Strided?
+  // LinearIndexing mean each thread reads consecutive elements.
```

Comment:
"means", "reads", "takes", "conserves", probably more readable if the comment for each enum value is on top of it.